docker/1.4.0/final/Dockerfile.eia

```diff
@@ -0,0 +1,36 @@
+ARG py_version
+
+FROM awsdeeplearningteam/mxnet-model-server:base-cpu-py$py_version
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends \
+    libopencv-dev \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /
+
+COPY dist/sagemaker_mxnet_container-2.0.0.tar.gz /sagemaker_mxnet_container-2.0.0.tar.gz
+
+RUN pip install --no-cache https://s3.amazonaws.com/amazonei-apachemxnet/amazonei_mxnet-1.4.0-py2.py3-none-manylinux1_x86_64.whl \
+                           keras-mxnet==2.2.4.1 \
```

Comment:
yeah check again with MXNet team if it makes sense to have keras-mxnet for any inference container at all.

Reply:
spoke with someone from the mxnet team. keras-mxnet has support for inference. so we will leave it in.